.github/issue_template.md

```diff
@@ -33,4 +33,4 @@ This issue is related to running on which platform? (Select all that apply)
 
 ## Minimal reproducible code (Required for bugs)
 
-Include a minimal code that produces the problem, add it inline using code block markdown, or a link to github repository:
+Include a minimal code that produces the problem (add it inline using code block markdown), or a link to Github repository:
```

Comment:
Produces -> Reproduces
Since the part within the parenthesis is connected to the part after the comma either all of it should be within the parenthesis or none of it.

Reply:
And if you want to capitalize github it should be GitHub.